draft-ietf-quic-multipath.md

```diff
@@ -455,23 +521,65 @@ the server MAY wait for a short, limited time such as one PTO if a path
 probing packet is received on a new path before sending the
 CONNECTION_CLOSE frame.
 
+Note that PATH_ABANDON frame is also used as a signal for the retirement 
+of the associated Path Identifier. When endpoint received PATH_ABANDON frame,
+it SHOULD not use the associated Path Identifier in future packets, it can 
+only use the Path ID in ACK_MP frames for inflight packets or 
+in MP_RETIRE_CONNECTION_ID frames for CID retirement.
```

Comment:
Should PATH_ABANDON automatically retire the connection IDs, without the need to send any frames? It would be nice if we don't have to deal with a malicious / misbehaving peer that abandons a path, but keeps the CIDs active.

Reply:
Yes, it should! Or at least the complete closing of a path, with dual `PATH_ABANDON` frames should.

Reply:
Record in #313